.travis.yml

```diff
@@ -25,7 +25,13 @@ jobs:
 
 jobs:
   include:
-    - if: type = pull_request
+    - stage: fast tests
```

Comment:
let's include the python version explicitly as 3.6 like below. stage name should be "Python 3.6: fail fast tests" since we're still running them in the other stages. also please include `- tox -e style --notests` in the install, otherwise install issues are hard to debug (and there shouldn't be any overhead)